tests/http-gap-beyond-body/test.yaml

```diff
@@ -0,0 +1,39 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+  min-version: 6.0.0
+
+# disables checksum verification
+args:
+  - -k none
+
+checks:
+
+  # Check that there is one file event with content range.
+  - filter:
+      count: 1
+      match:
+        event_type: http
+        http.url: "/1"
+        http.status: 200
+  - filter:
+      count: 1
+      match:
+        event_type: http
+        http.url: "/2"
+# response is not parsed in gap
+#       http.status: 200
+  - filter:
+      count: 1
+      match:
+        event_type: http
+        http.url: "/3"
+# response is no longer parsed after gap
```

Comment:
For now, we only recover from gaps with known size.

In the future, we could try to recover the 3rd response even if we missed the content-length of the second one